server/views/components/footer.html

```diff
@@ -15,7 +15,7 @@
         <a href="https://blockchainedindia.github.io/resources" target="_blank">Resources</a>
         <a href="/talks">Previous Talks</a>
     </div>
-    <form class="column subscribe" action="https://blockchainedindia.us15.list-manage.com/subscribe/post?u=43f23a03fbc4cf7179d9d1800&amp;id=de00b92d69" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
+    <form class="column subscribe" action="https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/subscribeform_sendy.html" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank" novalidate>
```

Comment:
Local routes doesn't work like this.

You need to add the route on nodejs express code and refer url route here.